File: WiTrackApp/WiTrackApp/Position.swift
//
//  Position.swift
//  WiTrack
//
//  Created by Jamar on 11/13/15.
//  Copyright © 2015 Jamar. All rights reserved.
//

import Foundation

class Position {
    var x : Float?
    var y : Float?
    var z : Float?
    var personId : Int?
    
    init(x : Float, y: Float, z: Float, personId : Int){
        self.x = x
        self.y = y
        self.z = z
        self.personId = personId
    }
}

File: WiTrack/app/src/main/java/util/Position.java
package util;

/**
 * Created by j388923r on 1/12/16.
 */
public class Position {
    public float x, y, z;
    public int personId;

    public Position(double x, double y, double z, int personId){
        this.x = (float)x;
        this.y = (float)y;
        this.z = (float)z;
        this.personId = personId;
    }
}
